app/controllers/application_controller.rb

```diff
@@ -267,7 +268,28 @@ def invalid_auth_token(_exception)
       user_signed_in: user_signed_in?,
     )
     flash[:error] = t('errors.general')
-    redirect_back fallback_location: new_user_session_url, allow_other_host: false
+    begin
+      redirect_back fallback_location: new_user_session_url, allow_other_host: false
+    rescue
+      analytics.unsafe_redirect_error(
```

Comment:
it's possible for Referer to hold an invalid redirect, but [rescue_from](https://edgeapi.rubyonrails.org/classes/ActiveSupport/Rescuable/ClassMethods.html#method-i-rescue_from) does not rescue if it's already rescued:

>Exceptions raised inside exception handlers are not propagated up.

Reply:
that is really good context, maybe that would make for a good comment in the code?

Reply:
Comment added 👍🏼